go.mod

```diff
@@ -1,6 +1,8 @@
 module github.com/rancher/charts-build-scripts
 
-go 1.18
```

Comment:
Shouldn't we just go ahead and upgrade to `1.23`?

Reply:
Just did it @tashima42